stock_available_product_expiry/reports/stock_scrap_expired.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2019 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <template id="stock_scrap_expired_document">
+        <t t-call="report.html_container">
+            <t t-foreach="docs" t-as="o">
+                <t t-call="stock.external_layout_barcode_right">
+                    <div class="page">
+                        <br/>
+                        <div class="row">
+                            <div class="col-xs-6">
+                                <div t-if="o.location_id.partner_id">
+                                    <span><strong>Location address:</strong></span>
+                                    <div t-field="o.location_id.partner_id" t-options='{"widget": "contact", "fields": ["address", "name", "phone", "fax"], "no_marker": True, "phone_icons": True}'/>
+                                </div>
+                            </div>
+                        </div>
+                        <br/>
+                        <h1 t-field="o.name" class="mt0"/>
+                        <table class="table table-condensed">
+                            <thead>
+                                <tr>
+                                    <th t-if="o.origin"><strong>Order (Origin)</strong></th>
```

Comment:
@acsonefho IMHO, should be 'Source Document' See https://github.com/OCA/stock-logistics-warehouse/blob/10.0/stock_available_product_expiry/models/stock_scrap_expired.py#L31